convert2rhel/main.py

```diff
@@ -168,26 +165,33 @@ def main_locked():
 
         rollback_changes()
 
-        report.summary(
+        report.pre_conversion_report(
             pre_conversion_results,
```

Comment:
Might as well add this
```suggestion
        report.pre_conversion_report(
            results=pre_conversion_results,
```

Reply:
updated.